.travis.yml

```diff
@@ -6,8 +6,12 @@ language:
 node_js:
 - '10'
 
+cache:
+  directories:
+    - ~/.npm
+    - node_modules
+
 install:
-- npm i -g npm
```

Comment:
Caching the NPM directories instead of installing npm globally improves Travis run time.